elf_reader.go

```diff
@@ -41,20 +41,30 @@ type ksymMeta struct {
 	Name    string
 }
 
+type structOpsSpec struct {
```

Comment:
I don't think this is necessary. `elfSection.relocations` already carries this information in the form of an `elf.Symbol`.

Reply:
right. this was removed.